bitnami/elasticsearch/values.yaml

```diff
@@ -254,6 +261,13 @@ coordinating:
   ## Number of coordinating-only node(s) replicas to deploy
   ##
   replicas: 2
+
+  ## updateStrategy for ElasticSearch coordinating deployment
+  ## ref: https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#update-strategies
```

Comment:
Link to deployments instead of stateful sets: https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy

Reply:
fixed!